CONTRIBUTING.md

```diff
@@ -0,0 +1,373 @@
+
+Please, don't use this repository for asking questions about Qiskit: there are
+better ways to do it. Refer to our public [Slack](https://ibm.co/joinqiskitslack)
+or post a question in [stack overflow](https://stackoverflow.com/questions/tagged/qiskit).
+
+## Before you start
+
+Contributing to qiskit.org, and other Qiskit repositories, assumes you have some level
+of [Git](https://git-scm.com) knowledge. For external contributors, a basic understanding
+of repositories, remotes, branches and commits is needed. For core contributors, you
+should know about resolving conflicts and rebasing too.
+
+There are tons of useful resources about Git [out there](https://try.github.io/).
+
+## Opening issues
+
+You can [open 4 types of issues](https://github.com/Qiskit/qiskit.org/issues/new/choose):
+
+* Bug reports: for reporting a misfunction. Provide steps to reproduce and expected behaviour.
+* Content issues: for noticing content-related problems such as typos or rewordings.
+* Tasks: a general "to do", in case none of the other templates suits you.
+* User stories: for identifying new value for the user.
+
+In addition to these 4 templates, the most common types of tasks are:
+
+* Plain tasks: these are the things that a core contributor usually work on during the sprint.
+* [Refactors](https://github.com/Qiskit/qiskit.org/issues?q=label%3A%22type%3A+refactor%22+): these aim at improving our codebase and reducing technical debt.
+* [Discussions](https://github.com/Qiskit/qiskit.org/issues?q=label%3A%22type%3A+discussion%22): these provide a central hub for talking about something to [reach a consensus](https://github.com/Qiskit/qiskit.org/issues?q=label%3ADecided+).
+
+Core contributors classify the tasks according to its nature and prioritize them
+from sprint to sprint. Types are not mutually exclusive and can change over time
+if needed.
+
+## Contributing code
+
+### Setup
+
+So you decided to get your hands dirty and start working on a patch? Then you
+need to know that qiskit.org follows the
+[Forking Workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/forking-workflow)
+with [Feature Branches](https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow).
+
+Regardless if you are a core contributor or not, the above means we expect you to fork
+the project on your own GitHub account and make your `master` branch to track this
+repository. A typical Git setup after
+[forking the project](https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github/fork-a-repo) is:
+
+```sh
+# After forking the repository in GitHub
+git clone https://github.com/<your_username>/qiskit.org.git
+cd qiskit.org
+git remote add upstream https://github.com/qiskit/qiskit.org.git
+git remote update upstream
+git checkout master
+git branch -u upstream/master
+git pull
+```
+
+### Working on an issue
+
+When you are going to start working on an issue, make sure you are in your `master`
```

Comment:
Maybe it is good to have here that when we start working on an issue, we assign ourselves and we move it to the "in progress column"

Reply:
We don't expect an external contributor will be part of sprint planning although this is definitely needed from a core contributor perspective. I'll do in the Wiki.